src/main/java/org/intellij/sdk/codesync/CodeSyncLogger.java

```diff
@@ -91,6 +93,9 @@ private static void logMessageToCloudWatch(
                 nextSequenceToken = error.expectedSequenceToken();
             } catch (DataAlreadyAcceptedException error) {
                 nextSequenceToken = error.expectedSequenceToken();
+            } catch (SdkClientException error){
```

Comment:
@Basit-qc Yes, here. Previously Saleem asked me to use ServiceUnavailableException because it was being thrown from putLogEvents() method but it did not work. I tried SdkClientException which was also being thrown and it worked. Also, it make sense considering its definition "SdkClientException – If any client side error occurs such as an IO related failure, failure to get credentials, etc.". I would wait for @saleem-latif review on this.